include/ddc/kernels/splines/spline_builder_batched.hpp

```diff
@@ -127,9 +180,17 @@ class SplineBuilderBatched
         return spline_tr_domain_type(bsplines_domain(), batch_domain());
     }
 
-    int offset() const noexcept
+    /**
+     * @brief Get the interpolation matrix.
+     *
+     * Get the interpolation matrix. This can be useful for debugging (as it allows
+     * one to print the matrix) or for more complex quadrature schemes.
+     *
+     * @return A reference to the interpolation matrix.
+     */
+    const ddc::detail::Matrix& get_interpolation_matrix() const noexcept
```

Comment:
@jbigot what do you think of returning the `ddc::detail::Matrix` ? Seems to be needed in gyselalibxx

Reply:
It would be good to merge and maybe we revert it later